2-dom-manipulation/form.js

```diff
@@ -0,0 +1,78 @@
+'use strict'
+
+const VAL_EMAIL = /^(([^<>()[\]\.,;:\s@\"]+(\.[^<>()[\]\.,;:\s@\"]+)*)|(\".+\"))@(([^<>()[\]\.,;:\s@\"]+\.)+[^<>()[\]\.,;:\s@\"]{2,})$/i;
+
+function validateForm(event){
+  event.preventDefault();
+
+  let name = document.querySelector('#contact_form').querySelector('#user_name'),  
+  email = document.querySelector('#contact_form').querySelector('#email_this'),
+  number = document.querySelector('#contact_form').querySelector('#user_phone_number'),
+  textBox = document.querySelector('#contact_form').querySelector('#text-box'),  
+  box =  document.querySelector('#contact_form').querySelectorAll('input[type="checkbox"]');
+  
+  validateName(name);
+  validarEmail(email);
+  validatePhone(number);
+  validateChecBox(box);
+
+  let valu = [name.value, email.value, number.value, validateChecBox(box), textBox.value];
+  let pu = [" Name:   ", "Email:   ", "Phone number:   ", "Courses:   ", "Message:   "];
+
+  for (let index = 0; index < valu.length; index++) {
+    for (let ind = 0; ind < pu.length; ind++) {
+      showInfotmation(pu, valu);      
+    }    
+  }
+};
```

Comment:
you don't need to do this, in Javascript you can use objects as dictionaries:

```
let formValues = {
  name:  'name',
   email:  'email'
};
// and then you can access these key/pair values

for (const property in formValues) {
  console.log(`${property}: ${formValues[property]}`);
}
```

In this way you don't need to do a double for loop which is performance heavy.